src/test/java/uk/gov/ch/developer/docs/controller/developer/SignOutControllerTest.java

```diff
@@ -39,87 +34,21 @@
     private HttpServletResponse httpServletResponse;
     @Mock
     private Session session;
+    @Mock
+    private Oauth2 oauth2;
```

Comment:
Why import this if its never used?

Reply:
We mocked Oauth2 so that when doSignOut is run during the test, the methods in Oauth2 are mocked which allows the test to run without error. I tried running the test without the mock, to see if it was needed but the results came back with a null pointer.

Reply:
Yeah, I just tried that as well. It's not stubbed explicitly because it doesn't return anything as part of the methods normal operation, we just need a mock in there so we can test the behaviour that comes afterwards

Reply:
Does it matter if it always returns null, should we be testing that only the correct methods are invoked?

Reply:
It can not reach the redirect if the mock is not there, because of the null pointer and for the test we are verifying if the sendRedirect is run